src/transformers/modeling_outputs.py

```diff
@@ -65,6 +65,21 @@ class BaseModelOutputWithNoAttention(ModelOutput):
     hidden_states: Optional[Tuple[torch.FloatTensor]] = None
 
 
+@dataclass
+class BaseModelOutputWithIntermediateActivations(ModelOutput):
+    """
+    Args:
```

Comment:
The Args: here should be below the first sentence

Reply:
Nice catch!